app/layout/home/carousel-section/component.tsx

```diff
@@ -36,12 +38,34 @@ export const HomeCarousel: React.FC<HomeCarouselProps> = () => {
 
   return (
     <div
-      style={{
-        background: 'linear-gradient(to bottom, #000 75%, #151515 25%)',
-      }}
+      className="relative"
     >
+      <div
+        role="presentation"
+        style={{
+          backgroundImage: `url(${BackgroundImage})`,
```

Comment:
Is there any reason to not using tailwind here? I understand that you can set the background image inline but the rest I think you can do it in tailwind.

Reply:
Done!